libs/components/src/next/form/select/vl-select.component.ts

```diff
@@ -115,6 +108,17 @@ export class VlSelectComponent extends FormControl {
             this.choices.config.removeItemButton = this.deletable;
             this.choices.config.removeItems = this.deletable;
         }
+
+        if (changedProperties.has('error')) {
+            (this.internals as ElementInternals).setValidity({ customError: this.error }, 'custom-error');
```

Comment:
kan dit niet bij de `FormControlMixin ` komen?

Reply:
Het is enkel de vl-select die dit moet doen, voor de andere componenten is deze fix niet nodig.